### PR TITLE
vsci: init at 7.0.12

### PR DESCRIPTION
**Pull request by D4ndellion**
###### Motivation for this change
vsci is a very popular cli video "contact sheet" generator.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by dali99** [comment]
@marsam you seem to do a lot of video related PRs

Requesting review ^-^

File: pkgs/tools/video/vcsi/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    jinja2
+    texttable
+    parsedatetime
+    nose
+  ];
+
+  postInstall = with stdenv.lib; ''
+    wrapProgram $out/bin/vcsi \
+      --prefix PATH : ${makeBinPath [ ffmpeg ]}
+  '';
```
Comment (jonringer) [inline pkgs/tools/video/vcsi/default.nix:26]: prevent double wrapping with wrapPython
```suggestion
  makeWrapperArgs = [ "--prefix PATH : ${lib.makeBinPath [ ffmpeg ]}" ];
```

File: pkgs/tools/video/vcsi/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, python3Packages, fetchFromGitHub, makeWrapper, ffmpeg }:
```
Comment (jonringer) [inline pkgs/tools/video/vcsi/default.nix:1]: not actually needed
```suggestion
{ lib, python3Packages, fetchFromGitHub, makeWrapper, ffmpeg }:
```

File: pkgs/tools/video/vcsi/default.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    parsedatetime
+    nose
+  ];
+
+  postInstall = with stdenv.lib; ''
+    wrapProgram $out/bin/vcsi \
+      --prefix PATH : ${makeBinPath [ ffmpeg ]}
+  '';
+
+  meta = with stdenv.lib; {
```
Comment (jonringer) [inline pkgs/tools/video/vcsi/default.nix:28]:
```suggestion
  meta = with lib; {
```

File: pkgs/tools/video/vcsi/default.nix (hunk 1 above)
Comment (dali99) [reply pkgs/tools/video/vcsi/default.nix]: I also removed wrapProgram from nativeBuildInputs

File: pkgs/tools/video/vcsi/default.nix [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{ lib, python3Packages, fetchFromGitHub, ffmpeg }:
```
Comment (marsam) [inline pkgs/tools/video/vcsi/default.nix:1]:
```suggestion
{ lib, python3Packages, ffmpeg }:
```

File: pkgs/tools/video/vcsi/default.nix [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    sha256 = "0dks0yr2a0cpr32vrwhdrhsb4qyj7rz1yv44fjbr8z8j8p84yjx5";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    numpy
+    pillow
+    jinja2
+    texttable
+    parsedatetime
+    nose
```
Comment (marsam) [inline pkgs/tools/video/vcsi/default.nix:18]:
```suggestion
```

Comment (dali99) [reply pkgs/tools/video/vcsi/default.nix]: no Idea where I got that from frankly.

**Review by marsam — APPROVED** [review]

**Pull request merged**